src/main/java/willy/task/EventsTask.java

```diff
@@ -0,0 +1,36 @@
+package willy.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A type of Task that spans over a period.
+ */
+public class EventsTask extends Task {
```

Comment:
Perhaps the class name should be consistent with the "DeadlineTask", like "EventTask"?

Reply:
Good point, thanks for reminding to be meticulous when naming the classes:)